src/Device.ts

```diff
@@ -197,4 +198,11 @@ class Device {
     }
 }
 
-new Device().start();
+
+const demo = async () => {
+    const cli = new Platform()
```

Comment:
Please let's stay simple and when we just add a cli parameter. No need to add interaction. This also breaks the current usage so far.